internal/config/nsmgr/nsmgr.go

```diff
@@ -88,11 +87,9 @@ func (mgr *NamespaceManager) NewPodNamespaces(cfg *PodNamespacesConfig) ([]Names
 	}
 
 	if len(cfg.Sysctls) != 0 {
-		pinnsSysctls, err := getSysctlForPinns(cfg.Sysctls)
-		if err != nil {
-			return nil, errors.Wrapf(err, "invalid sysctl")
+		for key, value := range cfg.Sysctls {
+			pinnsArgs = append(pinnsArgs, "-s", "%s=%s", key, value)
```

Comment:
"%s=%s" needs to be wrapped in single quotes, as sysctls can have spaces in them

Reply:
Isn't that preserved as we pass each "k=v" as distinct arguments?

Reply:
like `"'%s=%s'"`

Reply:
`/proc/sys/%s` > I had forgotten to format the `%s=%s` string before passing it as an argument, thus the `%s` in the error here.